lib/cc/analyzer/container_listener.rb

```diff
@@ -0,0 +1,22 @@
+module CC
+  module Analyzer
+    class ContainerListener
+      ContainerData = Struct.new(
```

Comment:
Didn't think too hard about it. `Container` seems like a good home too. `Container::ContainerData` is a little redundant, but that's probably OK.

I'd prefer not to do `Struct.new(...).new(...)` or use an `OpenStruct` at the call site.